mixlib-install.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.3"
   spec.add_development_dependency "pry"
+  spec.add_development_dependency "rb-readline"
   spec.add_development_dependency "vcr"
   spec.add_development_dependency "webmock", "~> 1.0"
+  spec.add_development_dependency "chefstyle", "~> 0.3"
```

Comment:
Out of curiosity, was there a specific reason for why we pinned `chefstyle` instead of using master?